locustfile.py

```diff
@@ -0,0 +1,73 @@
+"""
+Load testing script.
+
+Run via:
+
+$ locust
+
+Then visit http://localhost:8089 on host to run a test via the web UI.
+"""
+from locust import HttpLocust, TaskSet, task
+from hashlib import sha1
+from random import randint
+
+
+# NOTE: Change this based on your test user & plan
+USERNAME = 'testuser'
+PASSWORD = 'Test123$'
+PLAN_ID = 5
+
+# Configuration/instance specific. Assumes PA instance.
+NUM_DISTRICTS_IN_PLAN = 18
+REASSIGNMENT_POST_DATA = {
+    'geolevel': 2,
+    'geounits': 731,
+    'version': 0,
+}
+
+
+class UserActions(TaskSet):
+
+    def on_start(self):
+        self.login()
+
+    def login(self):
+        response = self.client.get('/accounts/login/')
+        csrftoken = response.cookies['csrftoken']
+        # Connection is unencrypted so the password is hashed before being sent
+        # to backend.
+        hashed_password = sha1(PASSWORD.encode('utf-8')).hexdigest()
+        self.client.post('/accounts/login/',
+            {
+             'username': USERNAME,
+             'password': hashed_password,
+             'next': '/districtmapping/plan/{plan_id}/edit/'.format(plan_id=PLAN_ID),
+            },
+            headers={'X-CSRFToken': csrftoken})
+
+    @task(1)
+    def index(self):
+        self.client.get('/')
+
+    @task(2)
+    def plan(self):
+        self.client.get('/districtmapping/plan/{plan_id}/edit/'.format(plan_id=PLAN_ID))
+
+    @task(10)
+    def reassign(self):
+        response = self.client.get('/districtmapping/plan/{plan_id}/edit/'.format(plan_id=PLAN_ID))
+        csrftoken = response.cookies['csrftoken']
+        # Reassign geounit to random district
+        self.client.post('/districtmapping/plan/{plan_id}/district/{new_district}/add/'.format(
```

Comment:
You could use Locust's naming feature here to ensure that these requests all get grouped under the same statistics: https://docs.locust.io/en/stable/writing-a-locustfile.html#grouping-requests-to-urls-with-dynamic-parameters